eidpdata/src/main/java/uk/ac/ucl/eidp/data/jaxb/JaxbSqlAnsi.java

```diff
@@ -36,12 +36,16 @@ protected String buildGetStatement() {
         stm.append(String.join(", ", queryFields));
         stm.append(" FROM ").append(tableType.getName());
         
-        if (!methodType.getFor().isEmpty()) stm.append(generateWhereClause());
+        if (!methodType.getFor().isEmpty()) {
+            stm.append(generateWhereClause());
+        }
              
         if (!methodType.getOrder().isEmpty()) {
             stm.append(" ORDER BY ");
             methodType.getOrder().forEach((MethodOrderType o) -> {
-                if (!stm.substring(stm.length() - 9).equals("ORDER BY ")) stm.append(", ");
+                if (!stm.substring(stm.length() - 9).equals("ORDER BY ")) {
+                    stm.append(", ");
+                }
```

Comment:
\[Checkstyle\] WARNING: 'if rcurly' have incorrect indentation level 16, expected level should be 14\.